components/menu/themes/spectrum-two.css

```diff
@@ -23,6 +23,6 @@
 		--spectrum-menu-item-focus-indicator-shadow: none;
 		--spectrum-menu-item-focus-indicator-offset: var(--spectrum-spacing-50);
 		--spectrum-menu-item-spacing-multiplier: 1;
-		--spectrum-menu-item-focus-indicator-outline-style: solid;
+		--spectrum-menu-item-focus-indicator-outline-style: none;
```

Comment:
Do we want to remove the focus indicator styles completely? The menu S2 token specs have the focus indicator, so is there a better solution? You may have already double checked with design and they're fine with its removal already!I wanted to mention it since it seems further away from S2 than I was expecting.

<img width="542" alt="Screenshot 2025-03-03 at 1 39 37 PM" src="https://github.com/user-attachments/assets/b1c52f81-409b-4f00-9bd5-d04a7897ed2c" />